test/bench/report_generator.py

```diff
@@ -182,15 +195,15 @@ def addVersionColumns(data):
     for row in space:
         try:
             splits = splitTagSemVer(row['tag'])
-            row['versionMajor'] = int(splits[0])
-            row['versionMinor'] = int(splits[1])
-            row['versionPatch'] = int(splits[2])
+            row['versionMajor'] = numericValueOf(splits[0])
+            row['versionMinor'] = numericValueOf(splits[1])
+            row['versionPatch'] = numericValueOf(splits[2])
             if len(splits) >= 4:
                 row['versionExtra'] = splits[3]
```

Comment:
`numericValueOf`? or at least comment explaining why not.

Reply:
Good point, the previous implementation assumed this field would always be text (alpha, beta...) but this isn't true. Now that the numeric value should now account for text sorting, I'm changing it to also sort based on an integer value as well.